qiskit/providers/ibmq/runtime/utils.py

```diff
@@ -64,3 +65,19 @@ def object_hook(self, obj: Any) -> Any:
                 decoded = base64.standard_b64decode(obj['__value__'])
                 return dill.loads(decoded)
         return obj
+
+
+class ProviderRequestParams(NamedTuple):
```

Comment:
Today one can find a runtime program by doing
```
sample_program = provider.runtime.program('sample-program')
```

The idea to this new enhancement is so that a user can also do something like 
```
inputs = sample_program.input_parameters()
```
which will return a `ProviderRequestParams` instance that has the program parameters as its attributes. This allows the user to do auto complete. Each program has its own set of input parameters so the attributes (and their docs) need to be created dynamically. 

The class also needs a more runtime-y name than `ProviderRequestParams` since this is runtime specific and the current name is too generic :).

Reply:
This change should be reflected now!